### PR TITLE
Modify access path of brick object.

### PR DESCRIPTION
**Pull request by nnDarshan**
This patch makes sure that brick object is
accessed in the modified location after the
path change.

tendrl-bug-id: Tendrl/gluster-integration#388
Signed-off-by: nnDarshan <darshan.n.2024@gmail.com>

### CODE REVIEW

**Comment by nnDarshan** [comment]
@shtripat pls review

**Comment by nnDarshan** [comment]
@shtripat @r0h4n @nthomas-redhat @anivargi Please review

File: tendrl/gluster_integration/objects/volume/atoms/generate_brick_mapping/__init__.py [hunk 1]
```diff
@@ -34,15 +34,22 @@ def run(self):
...
         )
-        for brick in bricks.leaves:
-            brick = brick.key.split("/")[-1]
-            _brick_host = brick.split(":")[0]
-            if _brick_host in nodes:
-                if len(nodes[_brick_host]) < brick_count:
-                    nodes[_brick_host].append(brick)
+        for host in hosts.leaves:
+            host = host.key.split("/")[-1]
+            bricks = NS._int.client.read(
```
Comment (shtripat) [inline tendrl/gluster_integration/objects/volume/atoms/generate_brick_mapping/__init__.py]: Cant we use load_all() here at least?

Comment (nnDarshan) [reply tendrl/gluster_integration/objects/volume/atoms/generate_brick_mapping/__init__.py]: The bricks we are trying to access here are free bricks, these are not objects but are references. So I feel load all can't be used here.

Comment (shtripat) [reply tendrl/gluster_integration/objects/volume/atoms/generate_brick_mapping/__init__.py]: Ack

**Pull request merged**